Automattic/portkey-android
# Removed untitled story logic.

Fixes #409

## Solution
Removed the logic that would set `Untitled` as the story title since the bottom sheet has been implemented. 

## Testing
1. Create a new story post. 
2. Click Publish. 
3. See that the upload notification says `null` as the title since it's not set. 

## Concerns

I am not sure if the demo app was supposed to implement some form of title setting so Ieft this as is. 

## Reviewing 
Only 1 reviewer is needed but anyone can review. 

## Submitter Checklist
